packages/blockchain-extension/extension/commands/addWalletIdentityCommand.ts

```diff
@@ -27,6 +27,19 @@ import { ManagedAnsibleEnvironment } from '../fabric/environments/ManagedAnsible
 import { EnvironmentFactory } from '../fabric/environments/EnvironmentFactory';
 import { LocalEnvironment } from '../fabric/environments/LocalEnvironment';
 
+export async function checkIdentityName(identityName: string, identityExists: boolean): Promise<boolean>
```

Comment:
No need to export it - we can just treat it like a 'private' function.
In our tests the function will just get called as normal when we're testing the `addWalletIdentityCommand` (no need to test `checkIdentityName` isolated).